src/utils/date.ts

```diff
@@ -1,3 +1,6 @@
+// TODO: Remove date-fns and covert to use @date-io/date-fns
+// https://github.com/dmtrKovalenko/date-io#projects
```

Comment:
This package looks like it still uses `date-fns` under the hood. Does it offer functionality we need for the filter?

Reply:
It's required for `<DatePicker>`'s `<LocalizationProvider>`.